### DISCUSSION TITLE
Objective evaluation about TTS

### DISCUSSION THREAD

**Discussion by inconnu11** [speechbrain/speechbrain#1660] [Q&A] [answered]
Hi, are there any objective evaluation scripts about evaluating the results of TTS systems?  For example, GPE, VDE,  or duration correlation.

**Comment by TParcollet** [comment]
@BenoitWang might know about that.

**Comment by BenoitWang** [comment, marked as answer]
Hi @inconnu11, sorry we don't have any objective evaluation metrics yet, seeing that most currently used evaluations remain subjective. However, it's possible that we add them in the future.
